src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -1167,6 +1183,38 @@ export class InlineChatController1 implements IEditorContribution {
 	get isActive() {
 		return Boolean(this._currentRun);
 	}
+
+	async createImageAttachment(attachment: string | URI): Promise<IChatRequestVariableEntry | undefined> {
+		const isImageFile = (path: string) => /\.(png|jpg|jpeg|gif|webp)$/i.test(path);
```

Comment:
Since it's `options.attachment` (which is good) we should be doing this the right way and support all kind of URI-based attachments. I know this extensions your scope but is the correct thing to do